src/domain.jl

```diff
@@ -82,10 +82,6 @@ function affect!(integrator, f::AbstractDomainAffect{T,S,uType}) where {T,S,uTyp
         dtcache = integrator.dt
         integrator.dt *= scalefactor
         dt_modified = true
-
-        # adjust new time step to bounds and time stops
-        fix_dt_at_bounds!(integrator)
-        modify_dt_for_tstops!(integrator)
         t = integrator.t + integrator.dt
 
         # abort iteration when time step is not changed
```

Comment:
Is there a way to reject the step and recompute it, in the same way as `isoutofdomain`, from the callback here? I think this would be preferable but IIRC it was not possible when I implemented the callbacks. Did this change, is it possible nowadays?